toolchain/lower/file_context.cpp

```diff
@@ -66,6 +70,22 @@ auto FileContext::Run() -> std::unique_ptr<llvm::Module> {
   return std::move(llvm_module_);
 }
 
+auto FileContext::BuildCompileUnit(llvm::StringRef module_name) -> void {
+  if (!include_debug_info_)
+    return;
+  // FIXME: Include directory path in the cu_file.
+  llvm::DIFile* cu_file = di_builder_.createFile(module_name, "");
```

Comment:
Ah - because it's the file for the compilation unit (cu) - I'll expand out that here. These domain-specific acronyms make more sense in other parts of LLVM/Clang that are more focussed just on debug info, but here where it's mixed with non-debug-info concepts, being more explicit is probably good.